src/_data/side-nav.yml

```diff
@@ -39,8 +39,6 @@
     - title: "Dart VM"
       permalink: /dart-vm
       children:
-      - title: "Install Dart"
```

Comment:
Where will people find this page?

Reply:
We have a **top-level sidenav entry named _Install_** under RESOURCES. Why should _Dart VM_ be singled out to have an install submenu item? If we have one for Dart VM, for sake of uniformity, we might as well have one for Web and for Mobile. Instead, I think that folks should be encouraged to go to the main install page in all cases. WDYT?

Reply:
Ah, I thought you were deleting the one under RESOURCES.